python/ray/ml/trainer.py

```diff
@@ -121,10 +123,11 @@ def training_loop(self):
             or a Callable that returns a Dataset, to use for training. If a
             ``preprocessor`` is also provided, it will be fit on this
             dataset and this dataset will be transformed.
-        extra_datasets: Any extra Datasets (such as validation or test
-            datasets) to use for training. If a ``preprocessor`` is
-            provided, the datasets specified here will only be transformed,
-            and not fit on.
+        datasets: Any Ray Datasets to use for training. Use
+            ``ray.ml.train.TRAIN_DATASET_KEY``. to denote which dataset is the training
+            dataset. If a ``preprocessor`` is provided and has not already been fit,
+            it will be fit on the training dataset. All datasets will be transformed
+            by the ``preprocessor`` if one is provided.
```

Comment:
Did we decide only the train dataset will be transformed? (fine if the decision is to try everything, just want to make sure this wasn't a typo)

Reply:
Hmm, I thought only train dataset will be fit, but everything will be transformed (validation dataset for example would need to be transformed still).

Reply:
Got it